cockatrice/src/dlg_update.cpp

```diff
@@ -163,6 +164,7 @@ void DlgUpdate::downloadSuccessful(QUrl filepath) {
     if (QDesktopServices::openUrl(filepath))
     {
         QMetaObject::invokeMethod((MainWindow*) parent(), "close", Qt::QueuedConnection);
+        qInfo() << "Opened downloaded update file successfully - closing Cockatrice";
```

Comment:
So this showing out of scope on the linux builds is a bug in QT < 5.4. Do we compile against that or lower for the linux builds?